cli/chain.go

```diff
@@ -1798,11 +1801,10 @@ var ChainInvokeEVMCmd = &cli.Command{
 				return nil
```

Comment:
It would be nice to do the same stripping of input data on invoke as you are doing for create: https://github.com/filecoin-project/lotus/pull/9942/files#diff-57b32822bc04a59777538e1130a95c9e0f19d0d12b04321a2a55803c6c5cffe5R1705